packages/size-limit/spinner.js

```diff
@@ -0,0 +1,70 @@
+const process = require('process')
```

Comment:
```suggestion
let process = require('process')
```

for the code style consistency